### DISCUSSION TITLE
Thuật ngữ "self-information"

### DISCUSSION THREAD

**Discussion by thanhcsf** [d2l-ai/d2l-vi#2975] [Terminology]
Thuật ngữ có trong issue #2933. (cc: @nguyenhoa93)

Trong Lý thuyết thông tin có thuật ngữ "self-information". Theo mình đọc, **"self-information"** hay còn gọi là **"information content", "suprisal", "Shannon information"**, được dùng để đo lường mức độ "bất ngờ" của một kết quả.
Mình có tham khảo một số nơi thì họ dịch thuật ngữ này là "Hàm lượng thông tin".
Mình không có kiến thức về Information Theory nên không chắc lắm. Nhờ các bạn cho ý kiến.

Tham khảo:
1. https://en.wikipedia.org/wiki/Information_content
2. https://thetalog.com/statistics/ly-thuyet-thong-tin/
3. https://link.medium.com/5pdyXRfhkab

**Comment by rootonchair** [comment]
Cái này lúc trước mình từng thảo luận rồi ạ https://github.com/aivivn/d2l-vn/pull/842
